### ISSUE TITLE
Need to add `add_metadata`

### ISSUE THREAD

**Issue by tomercervello** [m-chrome/py-suricataparser#8]
If metadata doesn't exist in rule, there should be an option to add_metadata. Trying `rule.add_option('metadata', 'a b')` fails.

**Comment by m-chrome** [comment]
I agree, it's good feature. Feel free to contribute changes you need.
